src/components/commit-list-item.component.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { StyleSheet, TouchableHighlight, View } from 'react-native';
+import { ListItem } from 'react-native-elements';
+import { colors, fonts } from 'config';
+
+type Props = {
```

Comment:
I think this should not go here since we're not using `@flow`. Thoughts @Antoine38660 @housseindjirdeh ?

Reply:
Nope quite a few components are using flow types :) I think it's definitely nice to have and hopefully we can increase type coverage throughout the app.

Reply:
But, we don't have `@flow` on top...so flow should not be checking anything. Or am I just missing something here?